examples/cfd/08_channel_flow.ipynb

```diff
@@ -0,0 +1,543 @@
+{
```

Comment:
<p>import numpy is redundant here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>